### PR TITLE
feat(release-please): add default branch input option

### PR DESCRIPTION
**Pull request by pragmaticivan**
Fixes https://github.com/google-github-actions/release-please-action/issues/201

### CODE REVIEW

**Comment by pragmaticivan** [comment]
Hum... the tests are passing locally with node 14.15.4. Investigating.

File: test/release-please.js [hunk 1]
```diff
@@ -46,6 +46,48 @@ describe('release-please-action', () => {
     })
   })
 
+  it('both opens PR to a different default branch and tags GitHub releases by default', async () => {
```
Comment (bcoe) [inline test/release-please.js:49]: Thank you for the test 🎉

File: test/release-please.js [hunk 2]
```diff
@@ -46,6 +46,48 @@ describe('release-please-action', () => {
...
+      tag_name: 'v1.0.0'
+    })
+    action.getGitHubRelease = () => {
+      class Release {}
+      Release.prototype.createRelease = createRelease
+      return Release
+    }
+    const releasePR = sinon.stub().returns(25)
+    const buildStatic = sinon.stub().returns({
+      run: releasePR
```
Comment (bcoe) [inline test/release-please.js:72]: Mind adding the inverse too, that confirms we default to `undefined` for the default branch? (at which point the default branch is currently detected).

**Review by bcoe — CHANGES_REQUESTED** [review]

**Comment by bcoe** [comment]
@pragmaticivan thank you for the contribution, one thought about a good additional test.

**Comment by pragmaticivan** [comment]
@bcoe just added a slight change on the test covering both. I felt it made more sense to complement the first test (default branch) as my change was a variation of the first one.

**Comment by pragmaticivan** [comment]
@bcoe I've also added this input into the readme. Missed that part.

**Review by bcoe — APPROVED** [review]

**Pull request merged**